Tools/Release/updateAMReX.py

```diff
@@ -102,7 +102,7 @@
 
 # Updates #####################################################################
 
-# run_test.sh (used also for Azure Pipelines)
+# run_test.sh (used also for Azure Pipelines) TODO: could maybe be short-cut now
```

Comment:
Cannot yet be removed because the regression test suite hard-codes that it needs AMReX early on.
This can be removed in a follow-up